website/docs/sources/x/index.md

```diff
@@ -1,45 +1,45 @@
 ---
-title: Twitter
+title: X
```

Comment:
I think for the title we should include the ` (formerly Twitter)` (both for SEO and also general find-ability)